layersvt/CMakeLists.txt

```diff
@@ -62,18 +77,31 @@ else()
     add_library(VkLayer_${target} SHARED ${ARGN})
     target_link_Libraries(VkLayer_${target} ${VkLayer_utils_LIBRARY})
     add_dependencies(VkLayer_${target} generate_api_cpp generate_api_h generate_api_html_h)
-    set_target_properties(VkLayer_${target} PROPERTIES LINK_FLAGS "-Wl,-Bsymbolic")
+    if (APPLE)
+        set_target_properties(VkLayer_${target} PROPERTIES LINK_FLAGS "-Wl")
+    else ()
+        set_target_properties(VkLayer_${target} PROPERTIES LINK_FLAGS "-Wl,-Bsymbolic")
+    endif ()
     install(TARGETS VkLayer_${target} DESTINATION ${CMAKE_INSTALL_LIBDIR})
     endmacro()
 endif()
 
-include_directories(
-    ${CMAKE_CURRENT_SOURCE_DIR}
-    ${CMAKE_CURRENT_BINARY_DIR}
-    ${CMAKE_BINARY_DIR}
-    ${Vulkan-ValidationLayers_INCLUDE_DIR}
-    ${JSONCPP_INCLUDE_DIR}
-)
+if (APPLE)
+    include_directories(
+        ${CMAKE_CURRENT_SOURCE_DIR}
+        ${CMAKE_CURRENT_BINARY_DIR}
+        ${CMAKE_BINARY_DIR}
+        ${Vulkan-ValidationLayers_INCLUDE_DIR}
+    )
+else ()
+    include_directories(
+        ${CMAKE_CURRENT_SOURCE_DIR}
+        ${CMAKE_CURRENT_BINARY_DIR}
+        ${CMAKE_BINARY_DIR}
+        ${Vulkan-ValidationLayers_INCLUDE_DIR}
+        ${JSONCPP_INCLUDE_DIR}
```

Comment:
I couldn't figure out what the JSONCPP issue was, but I think we can revisit when/if devsim gets ported

Reply:
Doesn't VIA use JSONCPP?  I know somebody else does, as that's why I used it for DevSim.

Reply:
It appears so, then fixing the JSONCPP include issue would be a prerequisite for porting either via or devsim to mac.